migration/i8.2z/oracle/202106281200_IDEMPIERE-4863.sql

```diff
@@ -0,0 +1,96 @@
+-- Table: AD_Wlistbox_Customization
+
+CREATE TABLE AD_Wlistbox_Customization
+(
+    AD_Client_ID 					NUMBER(10) NOT NULL,
+    AD_Org_ID 						NUMBER(10) NOT NULL,
+    AD_Wlistbox_Customization_ID 	NUMBER(10) NOT NULL, 
+    WlistboxName 					VARCHAR(255) NOT NULL,
+    AD_User_ID 						NUMBER(10) NOT NULL,
+    Created 						DATE DEFAULT SYSDATE,
+    CreatedBy 						NUMBER(10) NOT NULL,
+    Custom 							CHARACTER VARYING(2000) NOT NULL,
+    IsActive 						CHAR(1) DEFAULT 'Y' CHECK (IsActive IN ('Y','N')) NOT NULL, 
+    Updated 						DATE DEFAULT SYSDATE,
+    UpdatedBy 						NUMBER(10) NOT NULL,
+    AD_Wlistbox_Customization_UU 	VARCHAR2(36 CHAR) DEFAULT NULL,
+	CONSTRAINT AD_Wlistbox_Customization_Key PRIMARY KEY (AD_Wlistbox_Customization_ID),
+	CONSTRAINT AD_Wlistbox_Customization_UU UNIQUE (AD_Client_ID,AD_User_ID,WlistboxName),
+	CONSTRAINT AD_User_AD_WlistboxCustomization FOREIGN KEY (AD_User_ID) REFERENCES AD_user(AD_User_ID) DEFERRABLE INITIALLY DEFERRED	
+);
+
+
+CREATE UNIQUE INDEX ad_wlistbox_customization_uu_idx ON ad_wlistbox_customization (ad_wlistbox_customization_uu);
```

Comment:
The naming issue is still not address: ad_wlistbox_customization_uu_idx is for ad_wlistbox_customization_uu and AD_Wlistbox_Customization_UU constraint for (AD_Client_ID,AD_User_ID,WlistboxName). Should use a different name for the  AD_Wlistbox_Customization_UU unique constraint.

Reply:
Ok, done.